deployment/charts/cluster-manager/values.yaml

```diff
@@ -134,26 +135,21 @@ templateController:
   extraArgs:
     - "--webhook-enabled=true"
 
-
   extraEnv: []
 
   args:
     loglevel: "-loglevel=0"
     logformat: "-logformat=human"
 
-  metrics:
-    service:
-      # Whether to create a service for scraping metrics
-      enabled: false
-      port: 8080
-      # Labels to add to the service
-      labels:
+metrics:
+  enabled: false
```

Comment:
can this be override in the EMF ? if not is it possible to add a condition linked to "disable-metrics"?

Reply:
This can be overridden, it's implementation is the same as other feature gates